doc/cli_vs_yum.rst

```diff
@@ -99,6 +94,12 @@ following will work::
 
 ``include`` directive name of [main] and Repo configuration is a more logical and better named counterpart of ``exclude`` in DNF.
 
+=======================================================================
+``include`` option has been removed
+=======================================================================
```

Comment:
Though it probably doesn't matter, you can shorten the "====...." lines to match the length of the heading.